CoroutineTemplate/app/src/main/java/co/nimblehq/coroutine/ui/base/BaseNavigator.kt

```diff
@@ -42,36 +39,13 @@ abstract class BaseNavigatorImpl(
         findNavController()?.navigateUp()
     }
 
-    protected fun popBackTo(@IdRes destinationId: Int, inclusive: Boolean = false) {
-        findNavController()?.popBackStack(destinationId, inclusive)
-    }
-
     protected fun unsupportedNavigation() {
         val navController = findNavController()
         val currentGraph = activity.getResourceName(navController?.graph?.id)
         val currentDestination = activity.getResourceName(navController?.currentDestination?.id)
         handleError(NavigationError.UnsupportedNavigationError(currentGraph, currentDestination))
     }
 
-    protected fun NavController.navigateToDestinationByDeepLink(
-        destinationId: Int,
-        bundle: Parcelable? = null
-    ) {
-        createDeepLink()
-            .setDestination(destinationId)
-            .apply {
-                bundle?.let {
-                    setArguments(
-                        Bundle().apply {
-                            putParcelable("bundle", bundle)
-                        }
-                    )
-                }
-            }
-            .createPendingIntent()
-            .send()
-    }
-
```

Comment:
For template project, we won't need to focus on code coverage too strictly, instead, we could prepare some of the common things in our components. This is one of them, cleaning up is not necessary